.github/workflows/stale.yml

```diff
@@ -17,3 +17,6 @@ jobs:
         stale-pr-message: ' No activity on this PR for a prolonged duration, will be closed if no further activities in next 7 days'
         stale-issue-label: 'no-issue-activity'
         stale-pr-label: 'no-pr-activity'
+        exempt-issue-labels: 'enhancement, Feature, good first issue, help wanted, pending-product-review'
```

Comment:
re-check the labels, as I've changed their names, also add bug to the list

Reply:
Resolved in new commit.